kernel/src/main/java/org/kframework/kompile/Kompile.java

```diff
@@ -86,20 +87,32 @@
     java.util.Set<KEMException> errors;
 
     public Kompile(KompileOptions kompileOptions, FileUtil files, KExceptionManager kem, boolean cacheParses) {
```

Comment:
where is this constructor called? I would prefer not to construct an instance with an uninitialized field like this.

Reply:
this is being called from k-distribution/src/test/scala/org/kframework/backend/kore/KoreTest.scala line 41:
val compiler = new Kompile(options, files, kem, false)

Reply:
I guess one option is to initialize the outerParsingOption with kompileOptions.OuterParsing but I am not sure if we should be doing that. This constructor is called from a test and the other one is called from kompile so I'm tempted to say that this would be fine...